### ISSUE TITLE
Consider tag the latest release as prerelease

### ISSUE THREAD

**Issue by chenrui333** [libjpeg-turbo/libjpeg-turbo#488]
👋  Not sure if you folks are aware, github has [prerelase](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository#creating-a-release) concept, which might be more suitable for beta release.

Thanks!

![image](https://user-images.githubusercontent.com/1580956/103609046-f1254f00-4eea-11eb-86bd-ee35c3df383e.png)
![image](https://user-images.githubusercontent.com/1580956/103609096-131ed180-4eeb-11eb-860d-b8f4ec065c00.png)

**Comment by dcommander** [comment]
I'm aware, but I don't agree with their terminology.  I've been developing open source software professionally since 2004 and software in general since around 2000 (before that, I wore more of a hardware hat, since my actual degree is in Electrical Engineering.)

1. A beta release is an official release.  Calling it a "pre-release" is incorrect (a pre-release, by definition, has not been officially released) and would also be confusing, since [we already use the term "pre-release" correctly](https://libjpeg-turbo.org/DeveloperInfo/PreReleases).  Official releases of libjpeg-turbo (including beta releases) are built on my local lab machines using a pristine tarball system and signing keys that are not used for anything else (and whose private keys are carefully guarded on my local network.)  Thus, there is an extra layer of content security for official releases, whereas pre-releases are built using a CI system that runs on someone else's machines in the cloud.

2.  Saying that a beta release is not production-ready is either incorrect or redundant, depending on the situation.  For example, Red Hat Enterprise Linux 6 shipped with libjpeg-turbo 1.3 beta1 (1.2.90) throughout its entire life.  Their implementation was production-ready because they back-ported any bug fixes from later libjpeg-turbo releases into their package.  We already [explicitly define what "beta" means to our project](https://libjpeg-turbo.org/DeveloperInfo/PreReleases).  It doesn't necessarily mean "not production-ready", but it does tend to mean "hasn't been tested thoroughly in production environments yet."  Obviously, if no one ever considered the code ready to test in production environments, then it would never be tested in such environments, and it would never become production-ready.  Thus, with respect to libjpeg-turbo, "beta" conveys more of a subtle sense of "production-ready but probably not mission-critical just yet."

In short, I find the GitHub "pre-release" designation to be needlessly inflexible and have thus chosen not to use it.
